src/stylesheets/main.less

```diff
@@ -1,4 +1,28 @@
-body {
-  color: white;
-  background: black;
+@import "colors.less";
+
+body html {
+  width : 100%;
+  height: 100%;
+  margin: 0 auto;
+  display: flex;
+  flex-direction: column;
+  font-family: "Oxygen", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+  color: @text-dark;
+  background-color: @background;
+}
+
+@navbar-height : 63px;
+.title-bar{
```

Comment:
should we break out the title bar class to it's own css file? or do you prefer a monolith style file?